pants.travis

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Our integration tests invoke "./pants", and our regular runner script will trigger
+# a bootstrapping, which we don't want. So we shim in this alternative runner, which
+# calls the already-bootstrapped pants.pex.
+
+exec $(dirname "$0")/pants.pex "$@"
```

Comment:
Would it be possible to move this to build-support? I’m concerned about adding an additional file to the root, as it can be overwhelming for new Pants contributors.